build.js

```diff
@@ -0,0 +1,108 @@
+const execSync = require('child_process').execSync;
+const path = process.argv[2];
+const webcomponent = process.argv[3];
+
+class WebComponentBuild {
+	constructor(path, webcomponent) {
+		this.path = path;
+		this.webcomponent = webcomponent;
+	}
+
+	execute() {
+		this.__buildEs6();
+		this.__buildEs6Min();
+		this.__buildNode();
+		this.__commit();
+	}
+
+	__buildEs6() {
+		const e6BuildFile = `${this.path}/vl-${this.webcomponent}.js`;
+		this.__copySrcTo(e6BuildFile);
+		this.__maakStyleImportAbsoluut(e6BuildFile);
+		this.__maakDistImportsAbsoluut(e6BuildFile);
+		this.__maakSrcImportsAbsoluut(e6BuildFile);
+	}
+	
+	__buildEs6Min() {
+		const e6MinBuildFile = `${this.path}/vl-${this.webcomponent}.min.js`;
+		this.__copySrcTo(e6MinBuildFile);
+		this.__maakDistImportsAbsoluut(e6MinBuildFile);
+		this.__maakSrcImportsAbsoluut(e6MinBuildFile);
+		this.__inlineCss(e6MinBuildFile, this.path);
+		this.__minify(e6MinBuildFile);
+	}
+
+	__buildNode() {
+		const nodeBuildFile = `${this.path}/vl-${this.webcomponent}.src.js`;
+		this.__copySrcTo(nodeBuildFile);
+		this.__vervangWebcomponentenImportsDoorCommonJsImports(nodeBuildFile);
+		this.__vervangThidPartyImportsDoorCommonJsImports(nodeBuildFile);
+		this.__vervangLocalDistImportsDoorCommonJsImports(nodeBuildFile);
+		this.__vervangLocalSrcImportsDoorCommonJsImports(nodeBuildFile);
+		this.__inlineCss(nodeBuildFile, this.path);
```

Comment:
this.path hoeft niet doorgegeven te worden

Reply:
er is nog een ticketje om hier een automatische test van te maken ...